config/database.yml

```diff
@@ -9,16 +9,16 @@ development:
   <<: *default
   database: blog
   username: postgres
-  password: kiko
+  password: <%= ENV["BLOG_APP_DATABASE_PASSWORD"] %>
 
 test:
   <<: *default
   database: blog
   username: postgres
-  password: kiko
+  password: <%= ENV["BLOG_APP_DATABASE_PASSWORD"] %>
 
 production:
   <<: *default
   database: blog
   username: postgres
-  password: kiko
+  password: <%= ENV["BLOG_APP_DATABASE_PASSWORD"] %>
```

Comment:
- [ ] [OPTIONAL] To avoid this repetition, you may configure the username and password on the default config above, between lines 3 and 8. It is always great to minimize repetition as best as one can in order to make the work clean, understandable, scalable, and easy to make changes. :wrench: